ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateTransition.java

```diff
@@ -55,11 +54,7 @@ public void initiate(BeaconStateWithCache state, BeaconBlock block, Bytes32 prev
       blockProcessor(state, block);
     }
     // per-epoch processing
-    if (state
-        .getSlot()
-        .plus(UnsignedLong.ONE)
-        .mod(UnsignedLong.valueOf(SLOTS_PER_EPOCH))
-        .equals(UnsignedLong.ZERO)) {
+    if (state.getSlot() + 1 % SLOTS_PER_EPOCH == 0) {
```

Comment:
I believe this accidentally changed the order of operations. @cemozerr could you double check this?

```suggestion
    if ((state.getSlot() + 1) % SLOTS_PER_EPOCH == 0) {
```